api/swift/Sources/suborbital/lib.swift

```diff
@@ -44,19 +44,34 @@ public func Set(runnable: Runnable) {
     RUNNABLE = runnable
 }
 
-let httpMethodGet = Int32(1)
-let httpMethodPost = Int32(2)
-let httpMethodPatch = Int32(3)
-let httpMethodDelete = Int32(4)
+let httpMethodGet = Int32(0)
```

Comment:
```swift
enum HTTPMethod: Int32 {
    case GET
    case HEAD
    case OPTIONS
    case POST
    case PUT
    case PATCH
    case DELETE
}

public func HttpGet(url: String) -> String {
    return fetch(method: .GET, url: url, body: "")
}

public func HttpHead(url: String) -> String {
    return fetch(method: .HEAD, url: url, body: "")
}

public func HttpOptions(url: String) -> String {
    return fetch(method: .OPTIONS, url: url, body: "")
}

public func HttpPost(url: String, body: String) -> String {
    return fetch(method: .POST, url: url, body: body)
}

public func HttpPut(url: String, body: String) -> String {
    return fetch(method: .PUT, url: url, body: body)
}

public func HttpPatch(url: String, body: String) -> String {
    return fetch(method: .PATCH, url: url, body: body)
}

public func HttpDelete(url: String) -> String {
    return fetch(method: .DELETE, url: url, body: "")
}

func fetch(method: HTTPMethod, url: String, body: String) -> String {
```
This sparks a little more joy.

Reply:
@willemneal That's just Swift! You have to opt out of having a named parameter when you define the function if you don't want the caller to have to provide it.

Reply:
Ah I see.  And it seems to also have default parameters so you could remove some of the `body: ""`s.